from pytest import fail

def test_02 ():
    fail()